examples/raytrace-parallel/rust-toolchain.toml

```
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly"
+components = ["rust-src"]
+targets = ["wasm32-unknown-unknown"]
```

```suggestion
targets = ["wasm32-unknown-unknown"]
profile = "minimal"
```
Would this work as well?

---

I suppose we can give it a try...

The error we're getting is

```
error: 'cargo-clippy' is not installed for the toolchain 'nightly-x86_64-unknown-linux-gnu'
```

And `profile = "minimal"`  avoids installing clippy.

I don't see how not installing clippy will help us with a "clippy not installed" error. Could someone change the CI action to run `cargo +stable clippy`, or to install nightly clippy before checking the examples that require a nightly build?

---

Ah no, these two problems are unrelated. I was just asking if `profile = "minimal"` is enough to run this example, that's all!